scripts/hardhat/pfs-deploy.js

```diff
@@ -0,0 +1,188 @@
+const hre = require("hardhat");
+
+const witnet = require("../../witnet/assets")
+const network = hre.network.name
+const routes = require("../../witnet/routes/price")
+const utils = require("../utils")
+
+module.exports = { run };
+
+async function run(args) {
+
+  const selection = args.captions?.map(caption => {
+    return "Price-" + caption.toUpperCase()
+  }) || [];
+
+  const [ pfs, ] = await utils.getWitnetPriceFeedsContract();
+  
+  await settlePriceFeedsRadHash(pfs, selection)
+  await settlePriceFeedsRoutes(pfs, selection)
+
+  console.info()
+}
+
+async function settlePriceFeedsRadHash (pfs, selection) {
+  const addresses = witnet.getAddresses(network)?.requests;  
+  const prices = Object.fromEntries(
+    utils.flattenWitnetArtifacts(witnet.requests.price)
+      .map(value => [ value.key, value.artifact ])
+  )
+  for (const key in prices) {
+    const caption = utils.extractErc2362CaptionFromKey("Price", key)
+    try {
+      if (
+        !caption 
+          || (selection.length == 0 && utils.isNullAddress(addresses[key]))
+          || (selection.length > 0 && !selection.includes(caption))
+      ) {
+        continue;
```

Comment:
In general, I tend to avoid jump statements unless they are necessary and most of the [styleguides](https://eslint.org/docs/latest/rules/no-continue) disable them. What do you think about removing the `continue`?

Reply:
Yeah, but i'd rather prefer to state very clear the conditions under which requests declared within the project's assets won't be processed, and most of all avoid one more indentation level on the rest of the for loop, as it's already complex enough.